packages/react-components/lib/title-bar.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { createStyles, makeStyles, AppBar, Theme, Toolbar, Typography } from '@material-ui/core';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Badge from '@material-ui/core/Badge';
+
+export interface TitleBarProps {
+  logoPath: string;
+  tabNames: string[];
+  children?: React.ReactNode;
+}
```

Comment:
Maybe I'm missing something, how can the consuming component handle tab change events?

Reply:
My understanding is that the current tab change handler si currently 'overridden' by the dashboard URLs, so if we choose not to use URLs, then clicking on another tab will lead to an empty page with only just the header bar, and if we include a `<TabPanel/>`, the contents there will be displayed. like in the `material-ui` [demo](https://material-ui.com/components/tabs/#tabs)  Right now it just keeps the props in check, by changing the 'value' according to the individual tab values.

Reply:
The `children` are inside the `Toolbar`, so we can't add `TabPanel` this way, it would end up in part of the `Toolbar`. Also this creates some confusing API where the consumer must somehow know that the `TabPanel` values is the index of the `TabNames` props and that the first tab name must be the homepage link. Also, how can the consumer overwrite the tab behavior?

The most flexible would be a "NavigationBar" and "Toolbar" and let the consumer control it freely, an alternative is to take in a tab names + values props and a "onTabChange" props to let the consumer handle tab change behavior.